storagemarket/impl/provider.go

```diff
@@ -73,16 +82,17 @@ func NewProvider(net network.StorageMarketNetwork, ds datastore.Batching, bs blo
 	}
 
 	h := &Provider{
-		net:          net,
-		proofType:    rt,
-		spn:          spn,
-		fs:           fs,
-		pio:          pio,
-		pieceStore:   pieceStore,
-		conns:        connmanager.NewConnManager(),
-		storedAsk:    storedAsk,
-		actor:        minerAddress,
-		dataTransfer: dataTransfer,
+		net:                  net,
+		proofType:            rt,
+		spn:                  spn,
+		fs:                   fs,
+		pio:                  pio,
+		pieceStore:           pieceStore,
+		conns:                connmanager.NewConnManager(),
+		storedAsk:            storedAsk,
+		actor:                minerAddress,
+		dataTransfer:         dataTransfer,
+		dealAcceptanceBuffer: abi.ChainEpoch(100),
```

Comment:
NIT: can we put this in a constant at the top of the file? I prefer not to have hardcoded numbers in the constructor